internal/db/schema/migrations/postgres/2/05_job.up.sql

```diff
@@ -0,0 +1,79 @@
+begin;
+
+create table job (
+    id text primary key,
+    name wt_name not null,
+    description wt_description not null,
+    code text not null
+        constraint job_code_must_be_not_empty
+            check(length(trim(code)) > 0),
+    next_scheduled_run timestamp with time zone not null
+        default current_timestamp,
+
+    constraint job_name_code_uq
+        unique(name, code)
+);
+
+comment on table job is
+    'job is a base table where each row represents a unique job that can only have one running instance at any specific time.';
+
+create table job_name_enm (
+    name text not null primary key
```

Comment:
Is there a minimum or maximum length for `name`?

Reply:
Should I add a length check even if the enum is constrained to only have the allowed values?

Reply:
Oh this is weird the comment move from `job_name_enm` to `job_run_status_enm` after my changes to move them around.  I change `job_name_enum` to be a `wt_name` with checks already